src/VisualStudio/Core/Def/Implementation/Library/ObjectBrowser/AbstractObjectBrowserLibraryManager.cs

```diff
@@ -508,5 +512,51 @@ protected override bool TryExec(Guid commandGroup, uint commandId)
 
             return false;
         }
+
+        private IStreamingFindUsagesPresenter GetStreamingPresenter()
+        {
+            try
+            {
+                return _streamingPresenters.FirstOrDefault()?.Value;
+            }
+            catch
+            {
+                return null;
+            }
+        }
+
+        private async void FindReferences(
+            IStreamingFindUsagesPresenter presenter, SymbolListItem symbolListItem, Project project)
+        {
+            try
+            {
+                // Let the presented know we're starting a search.  It will give us back
+                // the context object that the FAR service will push results into.
+                var context = presenter.StartSearch(
+                    EditorFeaturesResources.Find_References, supportsReferences: true);
+
+                var cancellationToken = context.CancellationToken;
+
+                // Fire and forget the work to go get references.
+                var compilation = await project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
+                var symbol = symbolListItem.ResolveSymbol(compilation);
+
+                await AbstractFindUsagesService.FindSymbolReferencesAsync(
```

Comment:
:memo: `FindSymbolReferencesAsync` will throw a `NullReferenceException` if `symbol` is null.